docs/get-started/getting-started-alluxio.md

```diff
@@ -419,4 +419,93 @@ See the master.log and worker.log in this path.
 
 ### Auto start Alluxio the master and workers
 After installing Alluxio master and workers, it's better to add a systemd service for each process of master and workers.
-Systemd service can automatically start process if process is terminated.
\ No newline at end of file
+Systemd service can automatically start process if process is terminated.
+
+## Alluxio metrics
+### Monitor Alluxio metrics based on Alluxio Master Web
+Users can view the Alluxio metrics in the Web interface of Alluxio leading master:       
+http://<leading_master_host>:19999/metrics   
+For more details, please refer to section 3.1 of Alluxio doc: [Master Web UI Metrics](https://docs.alluxio.io/os/user/stable/en/operation/Metrics-System.html#default-http-json-sink)      
+The Alluxio Web UI is not available by default on databricks, the following provides a method to view the Web UI by SSH tunnel via jump server.   
+First forward the Alluxio port 19999 to a new port on a jump server, then forward the new port on the jump server to a local port, finally access the local port in the browser.   
+For example:   
+- Forward the port 29999 on jump server to local port 39999 on your own machine.   
+ssh -L 39999:localhost:29999 user@jump-server   
+- Forward the alluxio server 1999 port to the port 29999 on jump server.   
+ssh user@jump-server   
+ssh -L 29999:localhost:19999 alluxio_master_user@alluxio_master_host -p 2200 -i <private_key_file_path>   
+- Finally open http://localhost:39999/metrics on your own machine.   
+
+### Monitor Alluxio metrics based on Grafana and Prometheus
+Refer to section 3.2 of Alluxio doc: [Grafana Web UI with Prometheus](https://docs.alluxio.io/os/user/stable/en/operation/Metrics-System.html#default-http-json-sink)   
+Also the Web UI can be forward to local machine if the Alluxio Web UI is not available on databricks,   
+refer to the tunnel method in the previous section.   
+
+### View Alluxio historic metrics based on Grafana and Prometheus
```

Comment:
change to Grafana with Prometheus.  How is this section different from previous section?

Reply:
This is to view historic metrics. The previous section is to view live metrics on the Alluxio on Databricks.